.drone.yml

```diff
@@ -53,7 +53,7 @@ pipeline:
       # nodejs build is not in `docker-build` section,
       # because we have this deps:
       # build nodejs -> gulp build -> build python-prod
-      - docker-compose -f docker-compose-build.yml build nodejs
+      - docker-compose -f docker-compose-build.yml build --no-cache nodejs
```

Comment:
why we are needed no-cache there? PR code contains no changed js files

Reply:
I have created an issue for it https://github.com/fidals/shopelectro/issues/795